### ISSUE TITLE
documents and issues about save_server --template

### ISSUE THREAD

**Issue by adelezh** [xpdAcq/mission-control#186]
First, we could not find examples about how to run save_server with modified template. 

And we tried save_server - template='{base_folder}/{folder_prefix}/{start[analysis_stage]}/{start[sample_name]}_{human_timestamp}_{start[time]}_{start[original_start_uid]:.6}_{event[seq_num]:04d}{ext}'
the save_server window showed files are saving,  no errors or warnings, but we could not find the files at the destination folder.

**Comment by CJ-Wright** [comment]
Thank you for reporting!
Did you use ``save_server -- --help`` to look a the docs?
Did you find this page? https://xpdacq.github.io/xpdAn/xpdan.startup.html#save-server
(I'm trying to find out if the documentation we have is sufficient but difficult to find, insufficient but find-able or both hard to find and insufficient.)

**Comment by adelezh** [comment]
Yes, I found this page and followed the template, but it did not work well. 
It would be great if we can have a few examples.

    On Tuesday, May 21, 2019, 11:16:10 PM EDT, Christopher J. Wright <notifications@github.com> wrote:  
 
 
Thank you for reporting!
Did you use save_server -- --help to look a the docs?
Did you find this page? https://xpdacq.github.io/xpdAn/xpdan.startup.html#save-server
(I'm trying to find out if the documentation we have is sufficient but difficult to find, insufficient but find-able or both hard to find and insufficient.)
